### PR TITLE
Support all Kafka Admin client config

### LINKED ISSUE #37: Support extra Kafka client configuration
Currently the configurator only supports passing `bootstrap.servers` to the Kafka admin client. We should support all other [Kafka admin client config](https://kafka.apache.org/documentation/#adminclientconfigs)  (e.g. SSL config)

### PR DESCRIPTION
**Pull request by lacarvalho91**
Closes #37

### CODE REVIEW

**Comment by lacarvalho91** [comment]
I will add an example to the README before merging.

File: src/main/scala/com/sky/kafka/configurator/ConfigParsing.scala [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  def parse(args: Seq[String], envVars: Map[String, String]): Try[AppConfig] =
+    parser.parse(args, AppConfig()).map { appConfig =>
+      val providedConfig = appConfig.props ++ collectKafkaConfigFrom(envVars)
+      appConfig.copy(props = providedConfig.filterKeys(AdminClientConfig.configNames().asScala.contains))
+    }.toRight(InvalidArgsException).toTry
+
+  private def collectKafkaConfigFrom(envVars: Map[String, String]): Map[String, String] =
+    envVars.collect {
```
Comment (mludwinowiczsky) [inline src/main/scala/com/sky/kafka/configurator/ConfigParsing.scala]: there is no filtering here, map would be enough

File: src/main/scala/com/sky/kafka/configurator/ConfigParsing.scala [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      case (k, v) =>
+        k.split("KAFKA_").tail.mkString.replaceAll("_", ".").toLowerCase -> v
```
Comment (mludwinowiczsky) [inline src/main/scala/com/sky/kafka/configurator/ConfigParsing.scala]: when there is no "KAFKA_" element - you get ("", v) as a result,
when you get "ABCKAFKA_XYZ" env variable you treat it as  "KAFKA_XYZ" variable
is it intended? (probably it's no harm, just asking)

File: src/main/scala/com/sky/kafka/configurator/ConfigParsing.scala (hunk 1 above)
Comment (lacarvalho91) [reply src/main/scala/com/sky/kafka/configurator/ConfigParsing.scala]: good catch, i had filtering in there initially and removed it

File: src/main/scala/com/sky/kafka/configurator/ConfigParsing.scala (hunk 2 above)
Comment (lacarvalho91) [reply src/main/scala/com/sky/kafka/configurator/ConfigParsing.scala]: i think we can live with that

Comment (mludwinowiczsky) [reply src/main/scala/com/sky/kafka/configurator/ConfigParsing.scala]: ok

File: README.md [hunk 3]
```diff
@@ -94,7 +95,17 @@ Assuming you know the `<zookeeper_address>` and have placed your config file nam
...
 
+Any [Kafka Admin client config](http://kafka.apache.org/documentation/#adminclientconfigs) is supported, both when running the binary directly or via Docker. 
+
+These can be passed in the `--properties` command line option as key=value comma separated pairs:
+
+`kafka-configurator -f test-topics.yml --zookeeper localhost:2181 --properties client.id=foo,ssl.key.password=bar`
+
+Or they can be injected as environment variables prefixed with `KAFKA_`:
+
+`docker run -it -v <config_dir_on_host>:/etc/kafka-configurator -e KAFKA_CLIENT_ID=foo -e KAFKA_SSL_KEY_PASSWORD=bar skyuk/kafka-configurator -f=/etc/kafka-configurator/test-topics.yml --zookeeper=<zookeeper_address>`
```
Comment (josefelixh) [inline README.md]: ❤️

**Pull request merged**